misc/config_tools/schema/checks/cpu_assignment.xsd

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
Do you think we should also put a copyright and SPDX header comment in all our xsd files?

Reply:
Added to all XSD files for data checks. For the schema XSDs I'll update them the next time I update them (which will happen in a few days).